### ISSUE TITLE
Upgrade to BouncyCastle.Cryptography

### ISSUE THREAD

**Issue by pablopioli** [fszlin/certes#314]
BouncyCastle.Portable has been deprecated a while ago and replaced with BouncyCastle.Cryptography. 
https://github.com/bcgit/bc-csharp

Given how important updated cryptographic algorithms are I would suggest that an update to this library is needed.

I have tried to see how much effort is involved in this update, and found that only requires a couple of minor changes
https://github.com/pablopioli/certes/commit/00b7115cfc9145aa5f7ba367c6f2ef4e7dda29c9

A pull request for this will be useful?

**Comment by st-gwerner** [comment]
Yes please! It makes certes hard to use in cooperation with other packages, since the portable DLL has name conflicts with their official one. As you can imagine, this causes problems for people that use other packages that have updated their packages. (MimeKit comes to mind)

**Comment by fszlin** [comment]
Fixed in 4.0.0-beta.1: Certes now depends on `BouncyCastle.Cryptography` 2.7.0 instead of `Portable.BouncyCastle`. See the [changelog](https://github.com/fszlin/certes/blob/main/docs/CHANGELOG.md). Closing.
